.github/workflows/ci.yml

```diff
@@ -0,0 +1,88 @@
+name: CI
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  lint:
+    name: Linting Checks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup NodeJs
+        uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Run Linting Checks
+        run: yarn lint
+  test:
+    strategy:
+      fail-fast: true
+      matrix:
+        os:
+          - "ubuntu-latest"
+        node_version:
+          - "10"
+          - "12"
+          - "14"
+          - "15"
+        ts_version:
+          - "next"
+          - "latest"
+          - "3.4.1"
+          - "JS"
+    name: "Test - Node: ${{ matrix.node_version }} - TS: ${{ matrix.ts_version }} - OS: ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.ts_version == 'next' }}
+    env:
+      # Use compiled tests on everything instances except this one.
+      USE_COMPILED_TEST: ${{ fromJSON('["false", "true"]')[matrix.ts_version != 'latest' || matrix.node_version != '14' || matrix.os != 'ubuntu-latest'] }}
```

Comment:
Is there a nicer way to do this?
Using `fromJSON` as a ternary operator feels kinda hacky.